elements/pfe-navigation/src/pfe-navigation-item.scss

```diff
@@ -270,6 +270,7 @@ $variables: map-merge($variables, $nav-item-variables);
 :host([pfe-icon]) {
```

Comment:
I might suggest combining this code with the ::slotted selector above for clarity:

```
::slotted([slot="trigger"]) {
  @extend %trigger;
  :host([pfe-icon]) & {
    margin: 0.25em 0 0 0 !important;
    max-width: 100px;
  }
}
```